docs/api/07_coupons/apply-coupon.mdx

```diff
@@ -136,6 +149,9 @@ import TabItem from '@theme/TabItem';
 | coupon_code | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Code identifying the coupon.<br></br>It must match the `code` property of one of the active coupons. |
 | amount_cents | Integer &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Amount to apply to the customer.<br></br>If defined, it overrides the amount_cents property of the coupon |
 | amount_currency | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Currency of the amount to apply to the customer.<br></br>If defined, it overrides the amount_currency property of the coupon.<br></br>It must match the currency of the customer's plan. |
+| percentage_rate | Float &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Rate to apply with the coupon.<br></br>If defined, it overrides the percentage_rate property of the coupon |
+| frequency | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Frequency type.<br></br>It can be `once` or `recurring`.<br></br>If defined, it overrides the frequency property of the coupon |
+| frequency_duration | Integer &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | This field determines number of billing periods to which the coupon is applied.<br></br>If defined, it overrides the frequency_duration property of the coupon |
```

Comment:
This field determines the number of billing periods to which the coupon applies.

Reply:
Fixed grammar based on comments